locale/de_DE/LC_MESSAGES/djangojs.po

```diff
@@ -1027,42 +1019,42 @@ msgstr "Beantwortete Fragen"
 msgid "Affiliation Of Answered Questions"
 msgstr "Zugehörigkeit der beantworteten Fragen"
 
-#: meinberlin/apps/plans/assets/FilterNav.jsx:126
+#: meinberlin/apps/plans/assets/FilterNav.jsx:7
 msgid "Filter bar"
 msgstr "Filter Leiste"
 
-#: meinberlin/apps/plans/assets/FilterNav.jsx:127
+#: meinberlin/apps/plans/assets/FilterNav.jsx:8
 msgid "I am interested in projects from"
 msgstr "Mich interessieren Projekte aus"
 
-#: meinberlin/apps/plans/assets/FilterNav.jsx:167
-msgid "„ all Topics “"
-msgstr "„ alle Themen “"
-
-#: meinberlin/apps/plans/assets/FilterNav.jsx:183
-msgid "Which topic are you interested in?"
-msgstr "Welches Thema interessiert Sie?"
-
-#: meinberlin/apps/plans/assets/FilterNav.jsx:252
+#: meinberlin/apps/plans/assets/FilterNav.jsx:9
 msgid "„ all Districts “"
 msgstr "„ alle Bezirke “"
 
-#: meinberlin/apps/plans/assets/FilterNav.jsx:268
+#: meinberlin/apps/plans/assets/FilterNav.jsx:10
 msgid "Which district are you interested in?"
 msgstr "Welchen Bezirk suchen Sie?"
 
-#: meinberlin/apps/plans/assets/FilterNav.jsx:278
-msgid " in the area of "
-msgstr "und dem Bereich"
+#: meinberlin/apps/plans/assets/FilterNav.jsx:11
+msgid "„ all Topics “"
+msgstr "„ alle Themen “"
+
+#: meinberlin/apps/plans/assets/FilterNav.jsx:12
+msgid "Which topic are you interested in?"
+msgstr "Welches Thema interessiert Sie?"
 
-#: meinberlin/apps/plans/assets/FilterNav.jsx:319
+#: meinberlin/apps/plans/assets/FilterNav.jsx:13
 msgid "display projects"
 msgstr "Projekte anzeigen"
 
-#: meinberlin/apps/plans/assets/FilterNav.jsx:330
+#: meinberlin/apps/plans/assets/FilterNav.jsx:14
 msgid "more filters"
 msgstr "weitere Filter"
 
+#: meinberlin/apps/plans/assets/FilterNav.jsx:16
+msgid " in the area of "
+msgstr "und dem Bereich"
```

Comment:
Why do we need the spaces in the original string, but not in the translation? That's something that could be fixed with this PR! (If we don't need the spaces, also remove them in the original string.)